tests/unit_tests/test_collection_name_validation.py

```
@@ -38,27 +60,68 @@ def test_valid_names(self):
             "MyCollection_123",
             "A" * max_len,
         ]
+        print(f"\n{'=' * 60}")
+        print(f"测试: test_valid_names - 测试有效名称")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)